scripts/replace-cli-version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+VERSIONS=$(gh api -H "Accept: application/vnd.github+json" /repos/github/codeql-cli-binaries/releases | jq -r '.[].tag_name' | head -2)
+
+# we are exporting these variables so that we can access these variables in the workflow
+LATEST_VERSION=$(echo $VERSIONS | awk '{ print $1 }')
+PREVIOUS_VERSION=$(echo $VERSIONS | awk '{ print $2 }')
+
+echo "LATEST_VERSION=$LATEST_VERSION" >> $GITHUB_ENV
+echo "PREVIOUS_VERSION=$PREVIOUS_VERSION" >> $GITHUB_ENV
+
+sed -i "s/$PREVIOUS_VERSION/$LATEST_VERSION/g" supported_cli_versions.json
+sed -i "s/$PREVIOUS_VERSION/$LATEST_VERSION/g" extensions/ql-vscode/src/vscode-tests/ensureCli.ts
```

Comment:
It can probably ready from that, I didn't touch it because I've been able to run the project directly in my local 😂 which was why I asked yesterday to help me with the setup but didn't get time 😓

Reply:
Happy to help if you have setup problems.

Reply:
Added an [issue](https://github.com/github/code-scanning/issues/8598) to track this.